### DISCUSSION TITLE
关于SoraCommand中的MatchType

### DISCUSSION THREAD

**Discussion by hd80606b** [Hoshikawa-Kaguya/Sora#65] [Q&A] [answered]
MatchType中的KeyWord似乎并不是真正意义上的关键词匹配，而是只要句子里出现了这几个字就会触发
举例：使用指南里的例子，添加MatchType = Sora.Enumeration.MatchType.KeyWord 来匹配 坏耶
然而，你发送：“坏人太多了耶” ，“坏啊啊啊啊啊啊啊啊啊dwadaw啊啊啊啊啊啊耶啊啊啊2adwadwa” 
机器人也会回复 好耶
并不是常规的 string.contains 这种关键词匹配，请问是否是个bug

**Comment by Yukari316** [comment, marked as answer]
是bug，应该是正则写错了
已在新版本中修复 #66

**Reply by hd80606b** [reply]
感谢
